modules/post/windows/gather/enum_patches.rb

```diff
@@ -34,32 +34,33 @@ def initialize(info={})
 
   def run
     unless load_extapi
-      print_error 'ExtAPI failed to load'
+      print_error("ExtAPI failed to load")
       return
     end
 
     begin
-      objects = session.extapi.wmi.query("SELECT HotFixID FROM Win32_QuickFixEngineering")
+      objects = session.extapi.wmi.query("SELECT HotFixID, InstalledOn FROM Win32_QuickFixEngineering")
     rescue RuntimeError
-      print_error "Known bug in WMI query, try migrating to another process"
+      print_error("Known bug in WMI query, try migrating to another process")
       return
     end
 
     if objects[:values].nil?
       kb_ids = []
     else
-      kb_ids = objects[:values].reject(&:nil?).map { |kb| kb[0] }
+      kb_ids = objects[:values].reject(&:nil?).map { |kb| kb }
     end
 
     if kb_ids.empty?
-      print_status 'Found no patches installed'
+      print_status("Found no patches installed")
```

Comment:
See comment above for info on why these changes aren't really needed and actually break Ruby style convention.

Reply:
Fixed in next commit.